src/pages/home/components/FriendBanner.tsx

```diff
@@ -1,14 +1,21 @@
+import { useAuth } from "@/hooks/useAuth";
 import styled from "@emotion/styled";
 import { FiPlus } from "react-icons/fi";
 
 export const FriendBanner = () => {
+  const { user, isLoggedIn } = useAuth();
+
+  const text = isLoggedIn
+    ? `${user?.name}님, 선물할 친구를 선택해 주세요.`
+    : "선물할 친구를 선택해 주세요.";
+
   return (
     <OuterWrapper>
       <InnerBox>
         <IconBox>
           <FiPlus size={20} />
         </IconBox>
-        <Text>선물할 친구를 선택해 주세요.</Text>
+        <Text>{text}</Text>
```

Comment:
아주아주 사소한 부분이지만 중복되는 부분은 그냥 넣어줘도 괜찮을 것 같네용!

```suggestion
        <Text>{isLoggedIn && user ? `${user.name}님,` : ''} 선물할 친구를 선택해 주세요.</Text>
```